help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+
+- Destination is configured with the North America partition
+- Person with Email <test@example.com> and Company name Example Inc. In AEP matches the destination audience
+- Unless a record with those values already exists in the North America partition in Marketo, a new lead record will be created
+
+If no matching lead record is found, a new record will be created.
+
+![Destination Details](../../assets/catalog/adobe/marketo-engage-person-sync/destination-details.png)
+
+## Activate Audiences {#activate-audiences}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+In the Activate Audiences step, you will be able to select from any person audiences which are visible to you.
+
+![Activate Audiences](../../assets/catalog/adobe/marketo-engage-person-sync/activate-audiences.png)
+
+## Field Mapping {#field-mapping}
+
+For changes to a particular person attribute to be sent to Marketo Engage, the field must be mapped from a CDP field to Marketo Field.
+
+![Field Mapping](../../assets/catalog/adobe/marketo-engage-person-sync/field-mapping.png)
+
+AEP datatypes and Marketo datatypes can be mapped in the following ways:
+
+| AEP Data Type | Marketo Data Type                    |
+| ------------- | ------------------------------------ |
+| String        | String, Text Area, Url, Phone, Email |
+| Enum          | String                               |
+| Date          | Date                                 |
+| Date-time     | Datetime                             |
+| Integer       | Integer                              |
+| Short         | Integer                              |
+| Long          | Float                                |
+| Double        | Currency, Float, Percent             |
+| Boolean       | Boolean                              |
+| Array         | Not supported                        |
+| Object        | Not supported                        |
+| Map           | Not supported                        |
+| Byte          | Not supported                        |
+
+{style="table-layout:auto"}
+
+In some cases it is desirable to allow integrations to set the value of a field if there is none, while preventing integrations from making updates to fields which already have a value.  If you need to prevent the destination connector from overwriting existing values in your Marketo Engage instance, you can configure fields to block updates in the Admin->Field Management section of your Marketo instance and toggling the Adobe Experience Platform source type.
```

Comment:
Let's also add a section about validating data export, similar to the template. How do I tell if the data export has made it through successfully to Marketo?

https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#exported-data